react-app/ccf/src/pages/create-acc-applicants/CreateAccApplicant.css

```
@@ -286,3 +255,25 @@ select.input:focus {
   outline: none;
   border-color: #be0019;
 }
+
+.login-form {
+  justify-content: flex-start; 
+  align-items: center; 
+  overflow-y: auto;             
+  padding-top: 30px;   
+  max-height: 80vh;
+  padding: 20px 40px;         
```

Stylelint (17.7.0)

[error] Overridden property "padding-top" by shorthand "padding" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)